src/test/java/com/flowpowered/nbt/RegionFileTest.java

```diff
@@ -0,0 +1,48 @@
+package com.flowpowered.nbt;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import com.flowpowered.nbt.regionfile.Chunk;
+import com.flowpowered.nbt.regionfile.RegionFile;
+
+public class RegionFileTest {
+
+	@Rule
+	public TemporaryFolder folder = new TemporaryFolder();
+
+	/**
+	 * Test reading the NBT data in a region file
+	 *
+	 * @throws URISyntaxException
+	 * @throws IOException
+	 */
+	@Test
+	public void testRead() throws IOException, URISyntaxException {
+		RegionFile file = new RegionFile(Paths.get(getClass().getResource("/r.1.3.mca").toURI()));
```

Comment:
Do we really need to include large binary test files in this project? There is a LOT of unknown data in a minecraft region file. This test does not check the integrity of any of that data, all it does is go "yay! It did not crash!".

The best way to test would be to use a custom region file (or, even better! generate one in the test!) with a known and verifiable NBT structure, empty chunks, the works! Than you can test the NBT when you read it and make shore thing is corrupted.